electron/app/components/SampleModal.tsx

```diff
@@ -16,26 +17,20 @@ import { Button, ModalFooter } from "./utils";
 import * as selectors from "../recoil/selectors";
 import * as atoms from "../recoil/atoms";
 import Filter from "./Filter";
+import { Body } from "./CheckboxGrid";
```

Comment:
I've usually been adding these as properties of the main component, e.g. `CheckboxGrid.Body = Body` in CheckboxGrid.tsx, which would allow accessing the Body component as `CheckboxGrid.Body` here, and avoid name conflicts if you want to access more than one component named Body. I'd prefer to be consistent with that. 

(I suppose `import * as CheckboxGrid from "./CheckboxGrid"` would be similar, although I don't think you could access the component itself by doing that, so it could be confusing.)